webapp/src/Controller/API/GeneralInfoController.php

```diff
@@ -286,6 +288,48 @@ public function getConfigCheckAction(): JsonResponse
         return $this->json($result, $aggregate);
     }
 
+    /**
+     * Get the flag for the given country
+     * @Rest\Get("/country-flags/{countryCode}/{size}.svg")
+     * @OA\Response(
+     *     response="200",
+     *     description="Returns the given country flag in SVG format",
+     *     @OA\MediaType(mediaType="image/svg+xml")
+     * )
+     * @OA\Parameter(
+     *     name="countryCode",
+     *     in="path",
+     *     description="The ISO 3166-1 alpha-3 code for the country to get the flag for",
+     *     @OA\Schema(type="string")
+     * )
+     */
+    public function countryFlagAction(string $countryCode, string $size): Response
+    {
+        // This API action exists for two reasons
+        // - Relative URL's are relative to the API root according to the CCS spec. This
+        //   means that we need to have an API endpoint for files.
+        // - This makes it that we can not return a flag if flags are disabled.
+
+        if (!$this->config->get('show_flags')) {
+            throw new NotFoundHttpException('country flag not found');
```

Comment:
Well, we might have found it, but it's not enabled :-)

Reply:
Well I didn’t want to expose this difference but if you think we should I can change it

Reply:
Yep, I think we should make this explicit to ease debugging.